packs/equipment/explosive-mine-major.json

```diff
@@ -11,7 +11,7 @@
         "containerId": null,
         "damage": null,
         "description": {
-            "value": "<p><strong>Activate</strong> <span class=\"action-glyph\">2</span> Interact</p>\n<hr />\n<p>When you Activate an explosive mine, you place it on the ground in a square within your reach, priming it to explode when a creature steps on that square. If you have time in advance to prepare, you can place the mine on the ground before activating it, and you or an ally can try to use Stealth to Conceal an Object before you Activate the mine. If you don't Conceal the mine, its position is obvious at a glance. Once activated, the mine is primed to explode when enough pressure is placed on the square where it's located. Typically, this occurs when a Small or larger creature moves onto the square, though it could happen if a creature intentionally places pressure on the square from a distance to safely trigger the mine. Once triggered, the mine explodes, dealing the listed fire damage to any creatures in the area of effect with a basic Reflex save.</p>\n<hr />\n<p>The mine deals @Damage[18d6[fire]] damage, in a @Template[emanation|distance:10]{10-foot emanation}, and a @Check[reflex|dc:38|basic|name:Reflex|traits:fire].</p>"
+            "value": "<p><strong>Activate</strong> <span class=\"action-glyph\">2</span> Interact</p><hr /><p>When you Activate an explosive mine, you place it on the ground in a square within your reach, priming it to explode when a creature steps on that square. If you have time in advance to prepare, you can place the mine on the ground before activating it, and you or an ally can try to use Stealth to Conceal an Object before you Activate the mine. If you don't Conceal the mine, its position is obvious at a glance. Once activated, the mine is primed to explode when enough pressure is placed on the square where it's located. Typically, this occurs when a Small or larger creature moves onto the square, though it could happen if a creature intentionally places pressure on the square from a distance to safely trigger the mine. Once triggered, the mine explodes, dealing the listed fire damage to any creatures in the area of effect with a basic Reflex save.</p><hr /><p>The mine deals @Damage[18d6[fire]|options:area-damage] damage, in a @Template[emanation|distance:10], and a @Check[reflex|dc:38|basic|name:Reflex|traits:fire|options:area-effect].</p>"
```

Comment:
```suggestion
            "value": "<p><strong>Activate</strong> <span class=\"action-glyph\">2</span> Interact</p><hr /><p>When you Activate an explosive mine, you place it on the ground in a square within your reach, priming it to explode when a creature steps on that square. If you have time in advance to prepare, you can place the mine on the ground before activating it, and you or an ally can try to use Stealth to Conceal an Object before you Activate the mine. If you don't Conceal the mine, its position is obvious at a glance. Once activated, the mine is primed to explode when enough pressure is placed on the square where it's located. Typically, this occurs when a Small or larger creature moves onto the square, though it could happen if a creature intentionally places pressure on the square from a distance to safely trigger the mine. Once triggered, the mine explodes, dealing the listed fire damage to any creatures in the area of effect with a basic Reflex save.</p><hr /><p>The mine deals @Damage[18d6[fire]|options:area-damage] damage, in a @Template[emanation|distance:10], and a @Check[reflex|dc:38|basic|traits:fire|options:area-effect].</p>"
```